api/types/client.go

```diff
@@ -330,6 +330,11 @@ type PluginRemoveOptions struct {
 	Force bool
 }
 
+// PluginEnableOptions holds parameters to enable plugins.
+type PluginEnableOptions struct {
+	Timeout int64
```

Comment:
`time.Duration` lets users specify intervals more granular than seconds (ns, us, ms), which is not required here. `int64` is fine with what we want, but to be consistent, I've updated it to an `int`. `*int` is comes handy only when `0` value and `nil` pointer mean different things. In this case, it is not required, since `0` value means no timeout, which is also the default.